tests/functional/openstackclient_webhook_test.go

```diff
@@ -68,4 +72,40 @@ var _ = Describe("OpenStackClient Webhook", func() {
 			))
 		})
 	})
+
+	It("calls placement validation", func() {
```

Comment:
This should in a different test file as this file is for the openstackclient webhook